tests/controller_tests/test_ingest.py

```diff
@@ -59,6 +59,7 @@ def test_ingest_endpoint(test_client, example_data_model_file):
     assert response.status_code == 200, f"{response.json}"
     assert response.json == {
         "detail": "Spreadsheet successfully uploaded",
+        "metadata": {},
```

Comment:
Is it possible for us to have a test where this returns programme and fund_type?

Reply:
We can't set the `reporting_round` to 3 or 4 because we don't have a test submission or unit tests mocks to be able to do this yet.

This makes testing this endpoint end to end like this difficult at the moment.. we can mock out `get_metadata` to return some values but that doesn't really test anything.

Reply:
For now, I'll add a test that isolates `get_metadata` to return some values